test/Makefile.am.inc

```diff
@@ -79,24 +79,21 @@ AM_JS_LOG_FLAGS = \
 	@JASMINE_REPORT_ARGUMENT@ \
 	--no-config \
 	$(NULL)
-LOG_COMPILER = $(top_srcdir)/test/gtester-mock-event-recorder.sh
-AM_LOG_FLAGS = -k --verbose
+LOG_DRIVER = env AM_TAP_AWK='$(AWK)' $(SHELL) $(top_srcdir)/tap-driver.sh
+LOG_COMPILER = test/tap-test.sh
 EXTRA_DIST += \
+	test/tap-test.sh
 	test/run-with-dbus \
```

Comment:
I _think_ we can also stop running the tests under the fake session bus, if webhelper is disabled. Optional, if you want to dig into this, otherwise we can handle it when we move webhelper to a separate repo.